src/prefect/orion/models/deployments.py

```diff
@@ -436,6 +437,10 @@ async def _generate_scheduled_flow_runs(
         n=max_runs, start=start_time, end=end_time
     )
 
+    tags = deployment.tags
+    if auto_scheduled:
+        tags = ["auto-scheduled"] + tags
```

Comment:
Is this worth writing a test for?  It doesn't seem like you used this logic for this PR but either way I agree your change makes for a much better design

Reply:
yeah I think this deserves a test, I used it for the previous implementation but left it